.pre-commit-config.yaml

```diff
@@ -52,13 +52,13 @@ repos:
         args: ["--filter-files"]
         exclude: ^backend/.*/migrations
 
-      - id: mypy-conda
-        name: mypy-conda
-        entry: mypy
-        language: conda
-        require_serial: true
-        types_or: [python, pyi]
-        exclude: ^backend/.*/migrations
+      # - id: mypy-conda
```

Comment:
Hmm probably uncomment this and if it continues to be an issue we can dig in further :smiley: